src/ext/ddtrace.c

```diff
@@ -89,9 +95,103 @@ static PHP_RINIT_FUNCTION(ddtrace) {
     zend_hash_init(&DDTRACE_G(class_lookup), 8, NULL, (dtor_func_t)table_dtor, 0);
     zend_hash_init(&DDTRACE_G(function_lookup), 8, NULL, (dtor_func_t)ddtrace_class_lookup_free, 0);
 
+    zend_string *filename = zend_string_init(DDTRACE_G(request_init_hook), strlen(DDTRACE_G(request_init_hook)), 0);
+
+    if (filename && ZSTR_LEN(filename)!= 0){
+        run_a_file(filename);
+    }
```

Comment:
Just need a little cleanup here. :)
```suggestion
    }
    zend_string_release(filename);
```